app/models/member.rb

```diff
@@ -29,8 +29,10 @@ class Member < ActiveRecord::Base
 
   delegate :authenticate, to: :authentication, allow_nil: true
 
-  validates :email, uniqueness: true, allow_nil: true
-  before_save { email.try(:downcase!) }
+  validates :email, uniqueness: true, allow_nil: true, on: :create
```

Comment:
@NealJMD @osahyoun why are we allowing email to be `nil`? is there any case where we're creating members with nil email addresses?